wingtips-zipkin2/src/main/java/com/nike/wingtips/zipkin2/util/WingtipsToZipkinSpanConverterDefaultImpl.java

```diff
@@ -0,0 +1,61 @@
+package com.nike.wingtips.zipkin2.util;
+
+import com.nike.wingtips.Span;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.TimeUnit;
+
+import zipkin2.Endpoint;
+
+/**
+ * Default implementation of {@link WingtipsToZipkinSpanConverter} that knows how to create the appropriate client/server/local annotations
+ * for the {@link zipkin2.Span} based on the Wingtips {@link Span}'s {@link Span#getSpanPurpose()}.
+ *
+ * @author Nic Munroe
+ */
+public class WingtipsToZipkinSpanConverterDefaultImpl implements WingtipsToZipkinSpanConverter {
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Override
+    public zipkin2.Span convertWingtipsSpanToZipkinSpan(Span wingtipsSpan, Endpoint zipkinEndpoint) {
+        String traceId = wingtipsSpan.getTraceId();
+        long startEpochMicros = wingtipsSpan.getSpanStartTimeEpochMicros();
+        long durationMicros = TimeUnit.NANOSECONDS.toMicros(wingtipsSpan.getDurationNanos());
+
+        return createNewZipkinSpanBuilderWithSpanPurposeAnnotations(wingtipsSpan, startEpochMicros, durationMicros, zipkinEndpoint)
+            .id(wingtipsSpan.getSpanId())
+            .name(wingtipsSpan.getSpanName())
+            .parentId(wingtipsSpan.getParentSpanId())
+            .traceId(traceId)
+            .build();
+    }
+
+    protected zipkin2.Span.Builder createNewZipkinSpanBuilderWithSpanPurposeAnnotations(
+        Span wingtipsSpan, long startEpochMicros, long durationMicros, Endpoint zipkinEndpoint
+    ) {
+        zipkin2.Span.Builder zsb = zipkin2.Span.newBuilder()
```

Comment:
Now that we don't have to build timestamps for sr/ss/cr/cs annotations, we can move most of this into the main `convertWingtipsSpanToZipkinSpan(...)` method, and just have a simple helper method for converting wingtips `SpanPurpose` -> zipkin `Kind`, yes/no?

You don't need to do that refactoring - I'm planning on doing some other cleanup and refactoring on top of what you've done (e.g. probably moving to supporting `zipkin2.reporter.Reporter`s directly). Just making sure I'm not missing something.